assets/src/edit-story/karma/_init.js

```diff
@@ -119,6 +119,21 @@ beforeEach(() => {
   document.body.appendChild(rootEl);
   const body = rootEl.querySelector('test-body');
 
+  let clientX = -100;
```

Comment:
I like this idea, but I think we should separate it for now. Could you pls post this snippet in the feedback doc we have? I'll see the best way to instrument it.

Reply:
Ok, I will remove it from here and post the idea.